scripts/lib.sh

```diff
@@ -12,3 +15,60 @@ query_big_sur() {
     fi
     return 1
 }
+
+ensure-venv() {
```

Comment:
These 3 functions are from `python.sh` which I'm duplicating. On the following pass I will delete `python.sh` (unless I get a related getsentry PR landed first).

Reply:
I've now deleted `python.sh` since https://github.com/getsentry/getsentry/pull/5312 will be landing today.